.github/workflows/test.yml

```diff
@@ -8,8 +8,13 @@ jobs:
     container:
       image: georust/geo-ci
     steps:
+      - name: Install Rust toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          profile: minimal
+          override: true
+          components: rustfmt, rust-src
       - name: Checkout repository
         uses: actions/checkout@v2
-      - run: cargo install cargo-all-features
-      - run: cargo build-all-features
-      - run: cargo test-all-features
+      - run: cargo test
```

Comment:
Was it too slow to run all the feature combinations?

I think we should do at least `cargo test --all-features` and `cargo test --no-default-features`

Reply:
Good point, although `no-default-features` will cause our examples to fail…

Reply:
I think there's a way to declare required features for examples. Testing with `--no-default-features` still seems desirable.

Reply:
Yeah, trying to find the details now instead of putting `cfg`s everywhere

Reply:
https://stackoverflow.com/a/61417700 doesn't look great, though.

Reply:
Oh nooo